polus-basic-flatfield-correction-plugin/src/basic.py

```diff
@@ -290,20 +287,20 @@ def _get_flatfield_and_reweight(X_k_A,X_k_E,X_k_Aoffset,options):
     XE = np.reshape(X_k_E,(options['size'],options['size'],-1))
     XE_norm = XE/np.tile(np.reshape(np.mean(np.mean(XA,0),0)+10**-6,(1,1,-1)),(options['size'],options['size'],1))
     XAoffset = np.reshape(X_k_Aoffset,(options['size'],options['size']))
-    
+
     # Update the weights
     weight = 1/(np.abs(XE_norm) + options['epsilon'])
     options['weight'] = weight * weight.size / np.sum(weight)
-    
+
     # Calculate the mean flatfield and darkfield
     temp = np.mean(XA,2) - XAoffset
     flatfield = temp/np.mean(temp)
     darkfield = XAoffset
     return flatfield, darkfield, options
 
-def _get_photobleach(D,flatfield,darkfield=None):
+def _get_photobleach(imgf_stack,flatfield,darkfield=None):
     """ Calculate the global effect of photobleaching for each image
-    
+
     Using the original data, flatfield, and darkfield images, estimate the total contribution of photobleaching
     to an image in a series of images.
 
```

Comment:
It looks like the descriptions for this and the inexact alm might have been swapped.